File: mario2.c
#include <stdio.h>

int main() {
    int vezes;
    printf("Quantas vezes?: ");
    scanf("%d", &vezes);

    for (int i = 1; i <= vezes; i++) {
        for (int j = 0; j < vezes - i; j++) {
            printf(" ");
        }
        for (int j = 0; j < i; j++) {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < i; j++) {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: Mario2.java
import java.util.Scanner;

public class Mario2{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("quantas vezes?: ");
        int vezes = scanner.nextInt();
        for(int i = 0; i< vezes+1; i++){
            String espacos = " ".repeat(vezes - i);
            String estrela = "#".repeat(i);
            System.out.println(espacos + estrela + " " + estrela);
        }
    }
}